### ISSUE TITLE
Error parsing segments in get-params

### ISSUE THREAD

**Issue by Antonio-Laguna** [vordgi/nimpl-getters#61]
```
urlPathname: `/(.)select-product`;
pagePath: `/(frontend)/(.)select-product/page`;
```

> As a urlPathname there should be a path without the intercepted part. Can you please specify in the issue which URL this route for (or which URL is expected for)?

So in this scenario, I have

`/(frontend)/(.)select-product/page.tsx`
`/(frontend)/select-product/page.tsx`

**Comment by vordgi** [comment]
I'm looking and can't understand why I didn't cover the tests on groups and intercepted routes right away! 😬

And again thank you! Added more tests. Thanks to you, this package has not only fixed bugs, but also became more stable!

Fixed in version 1.3.4!

**Comment by Antonio-Laguna** [comment]
@vordgi thanks! That's ok! It's a weird usecase! Glad this got covered now! 🙇
